sys/src/main/java/com/alibaba/nacos/sys/utils/InetUtils.java

```diff
@@ -219,6 +219,26 @@ public static boolean isIP(String str) {
         return matcher.matches();
     }
     
+    /**
+     * juege str is right domain.
+     *
+     * @param str nacosIp
+     * @return nacosIp is domain
+     */
+    public static boolean isDomain(String str) {
+        try {
+            String[] split = str.split(":");
+            if (split.length != 2) {
+                return false;
+            }
+            Integer.parseInt(split[1]);
+            InetAddress address = InetAddress.getByName(split[0]);
+            return address.isReachable(100);
```

Comment:
Is this too strict to judge?

this method need the input must can be parse to IP and can reach.

But we just need check the input string is a domain pattern

Reply:
yes, just check the domain can resolve